pkgs/by-name/ti/tic-80/package.nix

```
@@ -0,0 +1,143 @@
+{ lib
+, pkgs
+, stdenv
+, fetchFromGitHub
+, libGL
+, alsa-lib
+, libX11
+, xorgproto
+, libICE
+, libXi
+, libXScrnSaver
+, libXcursor
+, libXinerama
+, libXext
+, libXxf86vm
+, libXrandr
+, libxkbcommon
+, wayland
+, wayland-protocols
+, wayland-scanner
+, dbus
+, udev
+, libdecor
+, pipewire
+, libpulseaudio
+, libiconv
+, ...
+}:
+with pkgs;
+stdenv.mkDerivation rec {
+  pname = "tic-80";
+  version = "1.1.2837";
+
+  src = fetchFromGitHub {
+    owner = "nesbox";
+    repo = "TIC-80";
+    rev = "v" + version;
+    sha256 = "sha256-2hSaLWw57F19tSfgJvBgbqW52vCu8p/TmoybYzQEybE=";
+    fetchSubmodules = true;
+    leaveDotGit = true; # Required for TIC-80's version detection via CMake.
+  };
+
+  # To avoid the awkward copyright range of "2017-1980", which would be caused by the
+  # sandbox environment, hardcode the year of the release.
+  postUnpack = ''
+    YEAR_OF_LAST_COMMIT=$(git -C source log -1 --format=%ad --date=format:%Y)
+    sed -E -i \
+      -e "s|string\(TIMESTAMP VERSION_YEAR \"%Y\"\)|set(VERSION_YEAR \"$YEAR_OF_LAST_COMMIT\")|" \
+      "source/CMakeLists.txt"
+  '';
+
+  # Taken from pkgs/development/compilers/mruby; necessary so it uses `gcc` instead of `ld` for linking.
+  # https://github.com/mruby/mruby/blob/e502fd88b988b0a8d9f31b928eb322eae269c45a/tasks/toolchains/gcc.rake#L30
+  preBuild = "unset LD";
+
+  cmakeFlags = [ "-DBUILD_PRO=On" "-DBUILD_SDLGPU=On" ];
+  enableParallelBuilding = true;
+  dontStrip = true;
+  buildInputs = [
+    cmake
+    pkg-config
+    wayland-protocols
+    python3
+    ruby
+    rake
+    git
+  ] ++ dlopenBuildInputs;
+  dlopenBuildInputs = [
```

```suggestion
  buildInputs = [
```
and with the fix above, we can just name this the normal name